package.json

```diff
@@ -12,10 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
     "main": "build/index.js",
     "scripts": {
-        "start": "webpack-serve --config ./webpack.serve.config.js --open --port 8000",
+        "start": "webpack-serve --config ./webpack.serve.config.js --open",
+        "start-mac": "export NODE_OPTIONS=--openssl-legacy-provider && webpack-serve --config ./webpack.serve.config.js --open --port 8000",
         "validate-init": "python _validate_init.py",
         "prepublishOnly": "npm run validate-init",
-        "build:js": "export NODE_OPTIONS=--openssl-legacy-provider &&  ./node_modules/webpack/bin/webpack.js --mode production",
+        "build:js": "webpack --mode production",

```

Comment:
You sure you don't need openssl now?

Reply:
It didn't have the open SSL before, this has been added recently, it doesn't work in the dap with the export command

Reply:
add dotenv package and add it in the .env. This will be an issue depending on version of node

Reply:
Yes you do. Openssl was upgraded In 18+ but one or more packages use the old one by default. I can update the whole repo if required

Reply:
If you don't mind that would be really helpful